modules/videoio/src/cap_dshow.cpp

```diff
@@ -2936,6 +2936,21 @@ int videoInput::start(int deviceID, videoDevice *VD){
     DebugPrintOut("SETUP: Device is setup and ready to capture.\n\n");
     VD->readyToCapture = true;
 
+    // check for optional saving the direct show graph to a file
+    const char* graph_filename = getenv("OPENCV_DSHOW_SAVEGRAPH_FILENAME");
+    if (graph_filename) {
+        std::vector<WCHAR> wfilename(strlen(graph_filename) + 1, L' ');
+        size_t len = mbstowcs(&wfilename[0], graph_filename, strlen(graph_filename) + 1);
+        CV_Assert(len == strlen(graph_filename));
```

Comment:
> strlen(graph_filename)

This expression has been used 3 times here. We may want to cache this value in a separate variable.